test/functional/test_framework/comptool.py

```diff
@@ -345,7 +345,7 @@ def run(self):
                                 # Most likely, we delivered a header for this block
                                 # but never had the block to respond to the getdata
                                 if send_witness:
-                                    c.send_message(msg_witness_block(block))
+                                    c.send_message(msg_block(block))
```

Comment:
This whole `if..else` can be simplified to just `c.send_message(msg_block(block))`

Reply:
true

Reply:
I did change this, although I do consider it a sort-of pointless endeavour as we are going to merge 0.17 soonish and comptool goes away in there. Anyway, this file's going to go away but for the time being we work with it.